qpmodel/PhysicalNode.cs

```diff
@@ -1057,7 +1060,18 @@ public class PhysicStreamAgg : PhysicAgg
     {
         public PhysicStreamAgg(LogicAgg logic, PhysicNode l) : base(logic, l) { }
         public override string ToString() => $"PStreamAgg({child_()}: {Cost()})";
-
+        public override PhysicProperty RequiredProperty()
+        {
+            var exprlist = (logic_ as LogicAgg).groupby_;
+            if (exprlist is null) return null;
+            return new PhysicProperty(exprlist);
+        }
+        public override PhysicProperty SuppiedProperty()
+        {
+            var exprlist = (logic_ as LogicAgg).groupby_;
+            if (exprlist is null) return null;
+            return new PhysicProperty(exprlist);
```

Comment:
do you mean it supplies (after streamAgg) sorted attributes on group by keys? If so, return "PhysicProperty" is not clear in naming. You may want to derive a "SortOrderProperty" out of Property class for this purpose?